src/Components/RelatedCollectionsRail/__tests__/RelatedCollectionsRail.test.tsx

```diff
@@ -38,6 +38,19 @@ describe("CollectionsRail", () => {
     expect(component.text()).toMatch("From $200")
   })
 
+  it("does not render a carousel entry if it has no artworks", () => {
+    const collectionsCopy = clone(props.collections)
+    collectionsCopy.push({
+      slug: "jasper-johns-flags2",
+      headerImage: "http://files.artsy.net/images/jasperjohnsflag.png",
+      title: "Jasper Johns: Flags Part 2",
+      price_guidance: 1000,
+      artworksConnection: null,
```

Comment:
Ah, so in the case of this bug, `artworksConnection` is coming back as null altogether rather than with `edges: []`

Curious! But this looks like a good fix.